.github/workflows/CI.yml

```diff
@@ -14,6 +14,31 @@ on:
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
+  conda-build:
+    name: Conda Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup conda environment
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniconda-version: "latest"
+          python-version: 3.11
+          environment-file: conda/environment.yml
+          auto-update-conda: false
+          auto-activate-base: false
+          show-channel-urls: true
+
+      - name: Run conda build
+        shell: bash -el {0}
+        # For the build, these environment variables would usually be set based on repo tags,
+        # but they can be these defaults instead since we don't use the build
+        env:
+          GIT_DESCRIBE_TAG: test
+          GIT_DESCRIBE_NUMBER: 0
+        run: conda build . --no-anaconda-upload
+
```

Comment:
Worth getting this in here, but at some other point it would be good to have a way of keeping this in synch with the CD action. 

Making this create an artefact that the CD consumes and uploads is one possibility.

Maybe fix it in https://github.com/payu-org/payu/issues/398

Reply:
Good point. Will investigate this in that issue.